static/bidder-params/datablocks.json

```diff
@@ -9,11 +9,7 @@
         "type": "integer",
         "minimum": 1,
         "description": "Website Source Id"
-      },
-      "host": {
-        "type": "string",
-        "description": "Network Host to request from"
       }
   },
-  "required": ["host", "sourceId"]
+  "required": ["sourceId"]
```

Comment:
@jmayor  this might be a breaking change.

Should scenario where incoming auction request  still has `host`. Prebid uses fail fast strategy. 

For example, if incoming auction request mentions multiple bidders. One of them is datablocks and has `host` param. 

Here request will fail in validation phase and entire auction will get aborted. This seems unfair for other bidders who are part of auction.

Therefore, consider having some transition period before removing `host` param. Giving enough opportunity to prebid server hosts/ publisher to make necessary change.

Reply:
There is no issue with removing a required parameter. If a request is received with the value, that value will simply be ignored and Prebid Server will be happy. This becomes a problem for adding a new required field.

Reply:
@SyntaxNode that is what I assumed - we should be good to go

Reply:
The `host` parameter is already listed as optional. This is ok.

I'm going to assume this PR will be merged soon enough that the datablocks adapter will be part of PBS 2.0. Thanks for the work @jmayor

Reply:
@bretg Thanks for your patience